client/src/components/ProductDetails/Details.js

```diff
@@ -33,15 +33,21 @@ const Details = ({ product }) => {
   const classes = DetailsStyles();
   const state = useAuthState();
   const history = useHistory();
-  // console.log(state.user);
+  console.log(state);
+
   const AddToCart = async () => {
     const addingProduct = await axios.post(
       `${BASE_URL}/cart/${product._id}`,
-      state
+      state,
```

Comment:
We don't need the entire state here,please pass only the variables needed.

Reply:
yes you're correct.Thanks Shubhankar!